coordinator/main.go

```diff
@@ -122,3 +116,34 @@ func run() (retErr error) {
 
 	return eg.Wait()
 }
+
+func newGRPCServer(reg *prometheus.Registry, log *slog.Logger) *grpc.Server {
+	issuer := snp.NewIssuer(logger.NewNamed(log, "snp-issuer"))
+	credentials := atlscredentials.New(issuer, nil)
+
+	grpcUserAPIMetrics := grpcprometheus.NewServerMetrics(
+		grpcprometheus.WithServerCounterOptions(
+			grpcprometheus.WithSubsystem("contrast_userapi"),
```

Comment:
metrics name are now wrong, as server is used by both user and mesh api.

Reply:
Also remember that this needs to be updated in the docs when changed.

Reply:
Good catch! I think I also made a mistake below resulting in no metrics being registered at all. Let me check that and see if I can work around that.

Reply:
Addressed in 6e1c6b87081f251ea04b7f62d2241a62a8b87b45